pandas/core/indexes/multi.py

```diff
@@ -1775,22 +1775,44 @@ def droplevel(self, level=0):
 
     def swaplevel(self, i=-2, j=-1):
         """
-        Swap level i with level j. Do not change the ordering of anything
+        Swap level i with level j.
+
+        Calling this method does not change the ordering of anything.
```

Comment:
(I know it was already there, but now we are changing it) The ordering of what? Maybe just the 'values' ?

Reply:
Done.